TextFieldEffects/TextFieldEffects/HoshiTextField.swift

```diff
@@ -131,7 +131,7 @@ import UIKit
         inactiveBorderLayer.frame = rectForBorder(borderThickness.inactive, isFilled: true)
```

Comment:
Do we need this to be the opposite of first responder? Is this broken as well when using the example project?